modules/op-release-notes-1-14.adoc

```diff
@@ -326,3 +326,23 @@ With this update, {pipelines-title} General Availability (GA) 1.14.3 is availabl
 * Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher was unable to reconcile the pipeline runs because of a race condition between deletion of existing pipeline runs and processing new pipeline runs. Because of this issue, some pipeline runs could not be processed.  With this update, the {pac} watcher processes pipeline runs.
 
 * Before this update, when you used the `tkn pr logs -f` command to view the logs for a running pipeline, the command line utility stopped responding, even if the pipeline run completed successfully. With  this update, the `tkn pr logs -f` command properly displays the log information and exits.
+
+[id="release-notes-1-14-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.4
+
+With this update, {pipelines-title} General Availability (GA) 1.14.4 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-4_{context}"]
+=== Fixed issues
+
+* Before this update, a large number of error messages referencing `tekton-pipelines-webhook.ConversionWebhook` was sometimes logged. With this update, unneeded conversion webhook configuration for the `ClusterTask` and `StepAction` Custom Resource Definitions (CRDs) was removed, and such error messages are no longer logged.
+
+* Before this update, some configuration settings using `options` sections in the `TektonConfig` custom resource (CR) did not work, because a race condition happened if the same setting was configured using both the `options` section and another field in the `TektonConfig` CR. With this update, the settings work.
+
+* Before this update, the {pipelines-shortname} console plugin pod did not move to the node specified using the `nodeSelector`, `tolerations`, and `priorityClassName` settings. With this update, the {pipelines-shortname} plugin pod moves to the correct node.
+
+* Before this update, some error messages were logged without the proper context information. With this update, error messages contain the required information.
+
+* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} operator controller failed to function correctly when starting and stopping resources in new namespaces. With this update, the controller functions correctly.
+
+* Before this update, configuring the Horizontal Pod Autoscaler using the `options` section in the `TektonConfig` CR was disabled. With this update, Horizontal Pod Autoscaler configuration using the `options` section works correctly.
```

Comment:
@mramendi under the `options`, 
* until 1.14.3, HPA update is supported, only creation of new HPA is disabled due to a bug. [1]
* from 1.14.4 HPA creation enabled.

[1] - https://docs.openshift.com/pipelines/1.14/about/op-release-notes.html
![image](https://github.com/openshift/openshift-docs/assets/1004403/cb5ded60-07fc-48b1-9f7b-933b90e4db43)

Reply:
PTAL now.